app/views/layouts/application.html.erb

```diff
@@ -25,9 +25,7 @@
 
   <body class="mainstream <%= yield :body_classes %>">
     <div id="wrapper" class="<%= wrapper_class(@publication || @presenter) %>">
-      <% unless current_page?(root_path) %>
-        <%= render 'govuk_publishing_components/components/contextual_breadcrumbs', content_item: @content_item %>
-      <% end %>
+
```

Comment:
This was a mistake - only removed for dev purposes, it needs to be restored.

Reply:
Restored ✅